src/main/java/com/mraof/minestuck/data/tag/MinestuckItemTagsProvider.java

```diff
@@ -126,6 +126,7 @@ protected void addTags()
 		tag(BUGS).add(BUG_ON_A_STICK.get(), CHOCOLATE_BEETLE.get(), CONE_OF_FLIES.get(), GRASSHOPPER.get(), CICADA.get(), JAR_OF_BUGS.get());
 		tag(CONSORT_SNACKS).add(Items.COOKIE).addTag(BUGS);
 		tag(CREATIVE_SHOCK_RIGHT_CLICK_LIMIT).add(Items.CHORUS_FRUIT);
+		tag(UNREADABLE).add(CLIENT_DISK.get(), SERVER_DISK.get(), GUTTER_THUMB_DRIVE.get(), GUTTER_BALL.get(), CAPTCHA_CARD.get(), CUEBALL.get(), BLACK_QUEENS_RING.get(), WHITE_QUEENS_RING.get(), BLACK_KINGS_SCEPTER.get(), WHITE_KINGS_SCEPTER.get()).add(Items.DRAGON_EGG).add(Items.DRAGON_HEAD).add(Items.DRAGON_BREATH).add(Items.NETHER_STAR).add(Items.COMMAND_BLOCK).add(Items.COMMAND_BLOCK_MINECART).add(Items.CHAIN_COMMAND_BLOCK).add(Items.REPEATING_COMMAND_BLOCK).add(Items.END_CRYSTAL);
```

Comment:
Having captcha cards in this tag would remove one method of getting spare captcha cards for alchemy during the early game. Has this been considered properly?

Reply:
this was the decision that had come to after a discussion between caldw3ll, blank, and to my knowledge riotmode, and I. I'm honoring it although I don't remember the actual purpose behind why specifically the captcha card was to be made unreadable. as it stands I have it locked away as something i can give or take once the feature is ready for packaging and release

Reply:
Iirc we decided to make the captchalogue cards unreadable in order to emphasize their use as the “fuel” for alchemy, along with some other reasons related to the role of cruxite dowels in alchemy. I think the conversation was had in Homestuck-observations? Take this with a grain of salt btw, cause I could very well be misremembering this.

Reply:
Wait, it was actually in debate-and-discussion.

Reply:
Ah yes, making them purposefully more sparse in the early game for some gameplay purpose, instead relying on the recipe with raw cruxite and paper.
However that does make me wonder, while cruxite will always be available in lands through mining, I don't think the same is true for paper or sugar cane. That could be problematic if we expect all players to need to use that recipe.

Reply:
we could create another more accommodating recipe, but that raises the question of whether or not it's worth it considering how needlessly gated it might make alchemy feel.

Reply:
I think maybe we could give sugarcane/paper more ways to be obtained. Add alchemy combinations like seaweed && stick = Sugarcane? Stick || wheat = sugarcane? 
We could also make it so that you can substitute the 8 paper in the captchalogue card recipe with one of the playing-card items you can sometimes get from imp-dungeons. Or maybe we could make it so that you can craft captchalogue cards using item frames?

Reply:
:?? < i do think sugar cane and paper, and more items in general, having alchemy recipes and alternative ways to be obtained in the medium is a good idea. what i dont understand is making captcha cards unreadable. 
:?? < it doesnt gate access to them, just adds two more steps; alchemizing the ingredients in their crafting recipes. at that point, why not just eliminate the middle man of having to alchemize the ingredients first?

Reply:
i'm in agreement with candy. it doesn't do much for captcha cards to be made unreadable considering their composite parts are already generally inexpensive and can be made in bulk.

Reply:
I thought raw cruxite wasn’t alchemizable? Even if it is though, making the captchalogue card’s code unreadable at first provides a sense of progression to the player, by removing a redundant step once they progress far enough. It’s like how the jumper block extension removes certain redundant steps. Removing the redundancies is a reward for progression.

Reply:
agreed, but the thing i'm questioning is whether or not it's worth it to make the alchemy experience bumpier when the speed with which you get to the same place remains the same. if we make cards unreadable, I think we should also look for ways to slow down the player's ability to get them to then emphasize their general value as a standalone item to an even greater extent. This would also smooth out the pacing of alchemy if making cards through crafting becomes the one available method.

Reply:
I think the speed of the process as a whole being basically the same is a good thing. It means that the restriction won’t bother players all that much, but despite not being bothered by it too much, they’ll still get that little hit of dopamine once they find out how to get around it using the intellibeam laserstation.

Reply:
That’s fair.